src/functions/edi/inbound/handler.ts

```diff
@@ -71,6 +72,26 @@ export const handler = async (
       filterDestination(d, { usageIndicatorCode, release })
     );
 
+    const destinationsWithEdiSource = filteredDestinations.filter(
+      (fd) => fd.destination.type === "webhook" && fd.destination.includeSource
```

Comment:
wdyt about trying to move the payload management logic to `src/lib/destinations/webhook.ts` instead of having it in the handlers? 

my concern is that we support the use of `webhook` destinations elsewhere (error handlers, remote poller, etc.), so putting the logic here will result in inconsistent behavior. it might be better to always include a `source` or `input` value to the delivery code, and let the delivery code decide whether or not to include it.